src/Automation/Automation.Sdk/README.md

```diff
@@ -47,7 +47,7 @@ input-file:
   - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/softwareUpdateConfigurationMachineRun.json
   - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/account.json
   - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/webhook.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/main/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json
+  - https://github.com/sushil490023/azure-rest-api-specs/blob/feature-FixAutomation/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json
```

Comment:
```suggestion
  - https://github.com/sushil490023/azure-rest-api-specs/blob/feature-FixAutomation/specification/automation/resource-manager/Microsoft.Automation/stable/2022-08-08/runbook.json
```
Hi, we strictly require the swagger to be from the main branch of Azure/azure-rest-api-specs repo. If you are just experimenting or in early dev stage, please make the pull request a draft. Thanks.

Reply:
Hi @isra-fel I have updated swagger to be from the main branch of Azure/azure-rest-api-specs repo. 
Can you please review it now.